client/src/components/WorkspaceCard/index.tsx

```diff
@@ -41,13 +44,20 @@ interface ConnectedPointerType {
 interface WorkspaceType {
   blocks: any[];
   childWorkspaceOrder: string[];
-  connectedPointers: any;
+  connectedPointersOfSubtree: ConnectedPointerType[];
   id: string;
 }
 
 interface WorkspaceCardProps {
+  isChild: boolean;
```

Comment:
`isChild` doesn't feel quite right as a name. I don't have a great proposal myself, but think the relevant concept is "is this workspace the root node of the subtree we're showing?". (Feel free to leave as is if you can't think of something better.)